Gruntfile.js

```diff
@@ -135,11 +135,10 @@ grunt.initConfig({
 		}
 	},
 
-	// Remove the requireSpacesInsideParentheses override once everything is fixed
 	jscs: {
 		all: {
 			options: {
-				requireCapitalizedComments: null
+				config: ".jscsrc"
```

Comment:
No, I meant this line. The default `config: true` should work just fine. In my own limited testing it seemed like `jscs` wouldn't report any errors unless I set `config` explicitly. Based on our conversation elsewhere it sounded like there's a bug where jscs wouldn't report any errors at all when there are more errors than maxErrors.